scripts/python/SLiM_sims.py

```diff
@@ -15,20 +15,63 @@
 import slim_vcf2fasta_chrom
 
 
-def run_slim(N, bot, region, outpath, slim_path):
+def args():
+    """Parse and return command-line arguments
+
+    Returns:
+        Command-line arguments
+    """
+    parser = argparse.ArgumentParser(description='Run simulations in SLiM and convert VCFs to FASTA using Drosophila reference',
+                                     usage='python3.7 SLiM_sims.py [options]')
+    parser.add_argument('-n', '--pop_size', help='The desired population size', type=int, required=True)
+    parser.add_argument('-b', '--bot', help='The desired strength of the population bottleneck. Expressed as the proportion of the population sampled during the bottleneck. 1.0=No bottleneck', type=float, required=True)
+    parser.add_argument('-u', '--mutation_rate', help='Per base-pair mutation rate in scientific notation (e.g., 1e-8). (Default: 2.8e-9)', type=float, default=2.8e-9)
+    parser.add_argument('-rec', '--recombination_rate', help='Per base-pair recombination rate rate in scientific notation (e.g., 1e-8). (Default: 2.14e-6)', type=float, default=2.14e-6)
+    parser.add_argument('-s', '--slim_path', help='Path to SLiM script.', type=str, required=True)
+    parser.add_argument('-f', '--fasta', required=True,
+                        type=str, help='Reference sequence in FASTA format')
+    parser.add_argument('-r', '--region', required=True,
+                        type=str, help='samtools format region (1 index)')
+    parser.add_argument('-m', '--mut_mat', required=True,
+                        type=str, help='LDhelmet mut mat file')
+    parser.add_argument('-o', '--outpath', help='Path to which VCFs from SLiM should be written', type=str, required=True)
+    args = parser.parse_args()
+
+    N = args.pop_size
+    bot = args.bot
+    outpath = str(args.outpath) + 'N{0}_bot{1}/'.format(N, bot)
+
+    return N, bot, args.mutation_rate, args.recombination_rate, args.slim_path, args.fasta, args.region, args.mut_mat, outpath
+
+
+def run_slim(N, bot, mu, rec, region, outpath, slim_path):
+    """Run simulation using SLiM
+
+    Args:
+        N (int): Population size
+        bot (float): Proportion of population to sample for bottleneck
+        mu (float): Per-base-pair mutation rate
+        rec (float): Per-base-pair recombination rate
```

Comment:
We should probably have units here for recombination rate (e.g. cM/bp vs M/bp) - LDhelmet returns M/bp, for instance, but most rates are traditionally reported in cM/Mb. I don't remember what SLiM expects as input alas